spanner/api/Spanner.Samples.Tests/EnableFineGrainedAccessTest.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Iam.v1;
+using Google.Apis.Iam.v1.Data;
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class EnableFineGrainedAccessTest
+{
+    private const string ServiceAccountId = "SpannerFGACTestAccount";

```

Comment:
This needs to have some sort of GUID, else, when tests are exeucting at the same time in Windows and Ubuntu, they'll overlap.